polaris-react/stylelint/plugins/coverage/index.js

```diff
@@ -0,0 +1,10 @@
+const stylelint = require('stylelint');
+
+const namespace = require('./utils/namespace');
+const rules = require('./rules');
+
+const rulesPlugins = Object.keys(rules).map((ruleName) => {
+  return stylelint.createPlugin(namespace(ruleName), rules[ruleName]);
```

Comment:
**suggestion (non-blocking):** I'm not sure we need the `namespace()` abstraction. What do you think about just adding a `const ruleNamePrefix = '@shopify/coverage'` to this file and updating this line to ```createPlugin(`${ruleNamePrefix}/${ruleName}`)```